routes/movies.js

```diff
@@ -0,0 +1,22 @@
+const express = require('express');
+const csrf = require('csurf');
+
+const router = express.Router();
+const csrfProtection = csrf({ cookie: true });
+
+const db = require('../db/models');
+
+const asyncHandler = (handler) => (req, res, next) => handler(req, res, next).catch(next);
+
+router.get('/:id(\\d+)', asyncHandler(async(req, res) => {
+    const movieId = parseInt(req.params.id, 10);
+    const movie = await db.Movie.findByPk(movieId);
+    // const year = movie.releaseDate.toString().split(' ')[3];
+    const year = movie.releaseDate.getFullYear();
+    res.render('movie-profile', { title: 'Movie Profile', movie, year });
+}));
+
+// TODO: Status Route
+    // add button with drop down functionality (pug,css)
```

Comment:
If you plan on leaving in this TODO, please make sure you reference a Github issue for implementing the feature that needs the Status route.

Reply:
Removed.